packages/orchestration/test/facade.test.ts

```diff
@@ -0,0 +1,69 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { setUpZoeForTest } from '@agoric/zoe/tools/setup-zoe.js';
+import type { CosmosChainInfo } from '../src/cosmos-api.js';
+import { makeOrchestrationFacade } from '../src/facade.js';
+import type { Chain } from '../src/orchestration-api.js';
+import { commonSetup } from './supports.js';
+
+const test = anyTest;
+
+export const mockChainInfo: CosmosChainInfo = {
+  chainId: 'mock-1',
+  connections: {},
+  icaEnabled: false,
+  icqEnabled: false,
+  pfmEnabled: false,
+  ibcHooksEnabled: false,
+  allowedMessages: [],
+  allowedQueries: [],
+  stakingTokens: [{ denom: 'umock' }],
+};
+
+test('chain info', async t => {
+  const { bootstrap } = await commonSetup(t);
+
+  const zone = bootstrap.rootZone;
+
+  const { zcf } = await setUpZoeForTest();
+
+  const { registerChain, orchestrate } = makeOrchestrationFacade({
+    localchain: bootstrap.localchain,
+    orchestrationService: bootstrap.orchestration,
+    storageNode: bootstrap.storage.rootNode,
+    timerService: bootstrap.timer,
+    zcf,
+    zone,
+  });
+
+  registerChain('mock', mockChainInfo);
+
+  const handle = orchestrate('mock', {}, async orc => {
+    return orc.getChain('mock');
+  });
+
+  const result = (await handle()) as Chain<any>;
+  t.deepEqual(await result.getChainInfo(), mockChainInfo);
+
+  // FIXME allow re-registering. perhaps this should be called `provide`.
+  t.throws(() => registerChain('mock', mockChainInfo), {
+    message: '"chainInfos" already registered: "mock"',
+  });
+
+  // FIXME support running again in the same zone (as will happen after an upgrade)
+  t.throws(
+    () =>
+      makeOrchestrationFacade({
+        localchain: bootstrap.localchain,
+        orchestrationService: bootstrap.orchestration,
+        storageNode: bootstrap.storage.rootNode,
+        timerService: bootstrap.timer,
+        zcf,
+        zone,
+      }),
+    {
+      message:
+        'key "chainInfos" has already been used in this zone and incarnation',
```

Comment:
darn it; I thought `zone.mapStore('chainInfos', ...)` already had _provide_ semantics.